civis/resources/_resources.py

```diff
@@ -403,12 +403,12 @@ def parse_path(path, operations, api_version, resources):
     return class_name, methods
 
 
-def parse_swagger(swagger, api_version, resources):
-    """ Parse a swagger specifiction into a dictionary of classes
+def parse_api_spec(api_spec, api_version, resources):
+    """ Parse the Civis API specifiction into a dictionary of classes
```

Comment:
The word "specification" is misspelled. (Unrelated, I know. I think it's still worth correcting here.)